sdk/appconfiguration/app-configuration/README.md

```diff
@@ -175,6 +178,134 @@ async function run() {
 run().catch((err) => console.log("ERROR:", err));
 ```
 
+### Create and get a snapshot
+
+```javascript
+const appConfig = require("@azure/app-configuration");
+
+const client = new appConfig.AppConfigurationClient(
+  "<App Configuration connection string goes here>"
+);
+
+const key = "testkey";
+const value = "testvalue";
+const label = "optional-label";
+
+const newSetting = await client.addConfigurationSetting({
+  key,
+  value,
+  label
+});
+const newSnapshot = await client.addSnapshot("testsnapshot", {
+  key,
+  label
+});
+/*
+  {
+    name: "testsnapshot",
+    status: "provisioning",
+    statusCode: 202,
+    filters: [{ key: "testkey", label: "optional-label" }],
+    compositionType: "all",
+    created: "2023-02-07T22:24:41.000Z",
+    retentionPeriod: 2592000,
+    size: 0,
+    itemsCount: 0,
+    tags: {},
+    etag: "T6JtbpthSWZQCkxmiK1OW6s3jMP-ZRYQPAFYZNbf0gE",
+  }
+ */
+let retrievedSnapshot = await client.getSnapshot("testsnapshot");
+console.log("Retrieved snapshot:", retrievedSnapshot);
```

Comment:
nitpick: statements look better in snippets when they have a blank line between them